### PR TITLE
Remove typeRoots and filesGlob

### PR DESCRIPTION
**Pull request by oshimayoan**

### CODE REVIEW

**Review by darcien — APPROVED** [review]
🐑 it.

File: tsconfig.json [hunk 1]
```diff
@@ -12,11 +12,11 @@
     // "declarationMap": true,                /* Generates a sourcemap for each corresponding '.d.ts' file. */
     "sourceMap": true /* Generates corresponding '.map' file. */,
     // "outFile": "./",                       /* Concatenate and emit output to single file. */
-    "outDir": "/dev/null" /* Redirect output structure to the directory. */,
+    // "outDir": "./build" /* Redirect output structure to the directory. */,
     // "rootDir": "./src",                    /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */
     // "composite": true,                     /* Enable project compilation */
     // "removeComments": true,                /* Do not emit comments to output. */
-    // "noEmit": true,                        /* Do not emit outputs. */
+    "noEmit": true /* Do not emit outputs. */,
```
Comment (sstur) [inline tsconfig.json:19]: Oops, this line was totally messing with me! I kept running build and deploy (firebase functions) but no new output was being generated, so the same code from `lib` folder was being deployed over and over, but I didn't realize it. I was quite confused.

Comment (oshimayoan) [reply tsconfig.json:19]: @sstur Oops, sorry about that.

So what do you think about it? Should we:
1. Change the default into `false`
2. Just comment it again
3. Let it be but make sure to overwrite it from local config/rule when needed

cc @darcien

Comment (darcien) [reply tsconfig.json:19]: I think option 2 is preferable.
Right now we mostly use this config for RN project, which don't need the output file.

**Comment by sstur** [comment]
option one and two are effectively the same. I’m good with either of those.
Just make sure that all of the projects using this config get updated
explicitly

**Comment by sstur** [comment]
I think just make sure the react native setup guide and has an instruction
about turning in noEmit

**Comment by darcien** [comment]
> I think just make sure the react native setup guide and has an instruction about turning in noEmit

Aye, I'll update the guide.

**Pull request merged**